ToDo

```diff
@@ -0,0 +1,28 @@
+----- >>>> DONE #1 Add - Unfollow Option to be introduced
+----- >>>> DONE #2 Updating follow_count in userData dynamically.
+----- >>>> DONE #3 Suggestion algo in view Connections to be improvised based on hobbies/goals
+----- >>>> DONE #4 Messaging list to be integrated with CURRENT_TIMESTAMP() 
+#5 Learning to build chat engine
+  ----- >>>> DONE #5.1 Establish connection and talk b/w browsers
+  ----- >>>> DONE #5.2 Store these messages in MySQL server and displaying Chat History.
+   **#5.3 Optimization of UI
+ ----- >>>> DONE  #5.4 scrollTo down of chat
+  ---->>>OBSOLETE <<!!NOT REALLY REQURED!!>> **#5.5 Clicking on img would open profile.
+   #5.6 Integrating it with notification system.
+   #5.7 Changing time format to WhatsApp format.
+
+----- >>>> DONE #6 I-card pop-up for view-profile section
+#7 Scroll Cache problem to be resolved
+#8 Change the backend of the friend-request sending algo
+----- >>>> DONE #9 Major reframing of login/signup form validation
+                     ---> Not resetting the whole form on one error
+#10 Setting up separate chatrooms per hobby and feature of mentioning someone
+----- >>>> DONE #11 Displaying users who are online
+----- >>>> DONE #12 Displaying green button if online
+----- >>>> DONE #13 Completely revamping of Login and signup form.
+----- >>>> DONE #14 Designing the forms for better UI/UX
+----- >>>> DONE #15 Polishing of Login/Sigup backend
+#16 Logo Change
+   
```

Comment:
add this file in .gitignor

Reply:
Okay! But do I need to clone the repo again or shall I just add 'ToDo' to .gitignore?

Reply:
ni sirf add karne se file track karna band kar dega,phir github se hata dena isse